internal/cmd/flink/command_compute_pool_create.go

```diff
@@ -0,0 +1,83 @@
+package flink
+
+import (
+	"github.com/spf13/cobra"
+
+	flinkv2 "github.com/confluentinc/ccloud-sdk-go-v2-internal/flink/v2"
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+func (c *command) newComputePoolCreateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create <id>",
+		Short: "Create a Flink compute pool.",
+		Args:  cobra.ExactArgs(1),
+		RunE:  c.computePoolCreate,
+	}
+
+	pcmd.AddCloudFlag(cmd)
+	c.addRegionFlag(cmd)
+	cmd.Flags().Uint32("cfu", 1, "Number of Confluent Flink Units (CFU).")
+	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
+	pcmd.AddOutputFlag(cmd)
+
+	cobra.CheckErr(cmd.MarkFlagRequired("cloud"))
+	cobra.CheckErr(cmd.MarkFlagRequired("region"))
+
+	return cmd
+}
+
+func (c *command) computePoolCreate(cmd *cobra.Command, args []string) error {
+	cloud, err := cmd.Flags().GetString("cloud")
+	if err != nil {
+		return err
+	}
+
+	region, err := cmd.Flags().GetString("region")
+	if err != nil {
+		return err
+	}
+
+	cfu, err := cmd.Flags().GetUint32("cfu")
+	if err != nil {
+		return err
+	}
+
+	environmentId, err := c.Context.EnvironmentId()
+	if err != nil {
+		return err
+	}
+
+	environment, err := c.V2Client.GetOrgEnvironment(environmentId)
+	if err != nil {
+		return err
+	}
+
+	computePool := flinkv2.FcpmV2ComputePool{Spec: &flinkv2.FcpmV2ComputePoolSpec{
+		DisplayName: flinkv2.PtrString(args[0]),
+		Cloud:       flinkv2.PtrString(cloud),
+		Region:      flinkv2.PtrString(region),
+		Config:      &flinkv2.FcpmV2ComputePoolSpecConfigOneOf{FcpmV2Standard: &flinkv2.FcpmV2Standard{Kind: "Standard"}},
+		MaxCfu:      flinkv2.PtrInt32(int32(cfu)),
```

Comment:
why are we getting uint32 then converting to int32?

Reply:
I wanted to prevent users from inputting negative values, but on second thought, the backend should handle this!